rmgpy/statmech/conformer.pyx

```diff
@@ -442,7 +445,7 @@ cdef class Conformer(RMGObject):
                           
         else:
             
-            raise ValueError("option {0} unimplemented or non-existant".format(option))
+            raise ValueError("option {0} unimplemented or non-existing".format(option))
```

Comment:
Perhaps change this message to `Option {0} is invalid. Should be either 1, 2, or 3.`? Since only three options exist and all are implemented.